source/includes/_orders.md

```diff
@@ -10,24 +10,25 @@ The `total_amount` is set by the `items` and the `resources`. See section below.
 
 Attributes | Type | Description
 ---------- | ---- | -------
-**user** | `object` | [`User`](#users) associated with the order
-**total_amount** | `number` | Amount as a number with two decimals. e.g., 12.34
 **currency** | `string` | 3 letter ISO currency code as defined by [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217)
-**order_status** | `string` | Status of the Order _default is CREATED, additionally there are: PROCESSING, SUCCESS, CANCELLED and FAILED_
-**delivery_status** | `string` | Status of the delivery _default is PENDING, additionally there are: ACCEPTED, REJECTED, ON_THE_WAY, ARRIVED, DONE and READY_FOR_DELIVERY_
-human_id | `string` | Human readable ID that identifies the order easily. e.g. `3AG7UA`
-provider | `string` | Provider assigned to an order
+billing_address | `object` | [`Address`](#address) used for billing purposes
+deliveries | `array` | Used to store the delivery history for an order
+delivery_address | `object` | [`Address`](#address) used for delivery
+delivery_status | `string` | Status of the delivery _default is PENDING, additionally there are: ACCEPTED, REJECTED, ON_THE_WAY, ARRIVED, DONE and READY_FOR_DELIVERY_
+delivery_time | `object` | Expected arrival time for delivery, `timestamp` format
 items | `array` | List of items associated with an order
-resources | `array` | An array of [`resources`](#resources) associated with the order.
+human_id | `string` | Human readable ID that identifies the order easily. e.g. `3AG7UA`
+note | `string` | Field used to send notes between an user and a provider
+order_status | `string` | Status of the Order _default is CREATED, additionally there are: PROCESSING, SUCCESS, CANCELLED and FAILED_
+payments | `array` | List of [`Payment`](#payments) objects associated with order
+payment_method | `object` | [`Payment Methods`](#payment_methods)
+provider | `string` | Provider assigned to an order used to pay the order
 units | `number` | Number of unique products in an order
-total_quantity | `number` | Total number of products in an order
+user | `object` | [`User`](#users) associated with the order
```

Comment:
You don't need the user

Reply:
Hmm :thinking: it's just the description of what an Order model contains

Reply:
But you are right ;)